docs/2.-API-reference.md

```diff
@@ -62,6 +62,18 @@ Set the server.
 * **`host`**: Host of the server
 * **`port`**: Port of the server
 
+#### AsyncMqttClient& setSecure(bool `secure`)
+
+Whether or not to use SSL. Defaults to `false`.
+
+* **`secure`**: SSL wanted or not.
+
+#### AsyncMqttClient& setServerFingerprint(const uint8_t\* `fingerprint`)
```

Comment:
As you say, this method *adds* a fingerprint. Can you please rename this method to `addServerFingerprint`?